docs/core/tools/global-json.md

```diff
@@ -65,15 +140,30 @@ dotnet new globaljson --sdk-version 2.2.100
 > The matching rules are governed by the apphost, which is part of the .NET Core runtime.
 > The latest version of the host is used when you have multiple runtimes installed side-by-side.
 
-Starting with .NET Core 2.0, the following rules apply when determining which version of the SDK to use:
+## [.NET Core 3.x](#tab/netcore3x)
+
+Starting with .NET Core 3.0, the following rules apply when determining which version of the SDK to use:
 
-- If no *global.json* file is found or *global.json* doesn't specify an SDK version, the latest installed SDK version is used. Latest SDK version can be either release or pre-release - the highest version number wins.
+- If no *global.json* file is found, or *global.json* doesn't specify an SDK version nor an `allowPrerelease` value, the latest installed SDK version is used (equivalent to setting `rollForward` to `latestMajor`). Whether the latest SDK version can be release or prerelease depends on how `dotnet` is being invoked.
+  - If you're **not** in Visual Studio, prerelease versions are considered.
```

Comment:
```suggestion
  - If you're **not** in Visual Studio, prerelease versions are considered, unless `allowPrerelease` is set to false in *global.json*.
```

Reply:
I won't take this once since the condition specifies that allowPrerelease is not set in the previous sentence:

"If no *global.json* file is found, or *global.json* doesn't specify an SDK version nor an `allowPrerelease` value"